### PR TITLE
feat(plugin): Drawboard

### PR DESCRIPTION
**Pull request by MaiKokain**
A canvas board, allowing for adding texts/images over the canvas board, and more.

This could probably also complete https://github.com/Vencord/plugin-requests/issues/219

### CODE REVIEW

**Comment by TheKodeToad** [comment]
Disboard??? What does it do?

File: src/plugins/disboard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    return (
+        <ChatBarButton onClick={() => openModal(props => <BoardModal modalProps={props} />, { modalKey: "ExcaliFakeBoard", onCloseRequest: () => { console.log("Close requested"); closeModal("ExcaliFakeBoard"); } })} tooltip={""}>
+            <img src="https://raw.githubusercontent.com/TheOriginalAyaka/sekai-stickers/main/public/img/emutest.png" width={16} height={16} />
+        </ChatBarButton>
+    );
+};
+
+export default definePlugin({
+    name: "idkwhattonamethisye", // give rec should i just call it "Excaliboard"
+    description: "idk just a worser version of (not) excaliboard",
```
Comment (Scyye) [inline src/plugins/disboard/index.tsx:27]: sorry, what?

**Comment by MaiKokain** [comment]
i'll put description later my internet's f-ing with me rn

**Comment by Vendicated** [comment]
relax, it's marked as draft for a reason

**Comment by programminglaboratorys** [comment]
> relax, it's marked as draft for a reason

someone please buy me glasses

**Pull request closed without merging**